src/main/io/gps.c

```diff
@@ -954,24 +955,30 @@ void setSatInfoMessageRate(uint8_t divisor)
 void gpsConfigureNmea(void)
 {
     static bool atgmRestartDone = false;
-#if !defined(GPS_NMEA_TX_ONLY)
-#endif
+
     DEBUG_SET(DEBUG_GPS_CONNECTION, 4, (gpsData.state * 100 + gpsData.state_position));
+
     switch (gpsData.state) {
         case GPS_STATE_DETECT_BAUD:
-#if !defined(GPS_NMEA_TX_ONLY)
+#if !defined(USE_GPS_NMEA_TX_ONLY)
             if (cmp32(gpsData.now, gpsData.state_ts) < 1000) {
                 return;
             }
             gpsData.state_ts = gpsData.now;
             switch (gpsData.state_position) {
                 case 0: // first run after bootup
-                    serialSetBaudRate(gpsPort, 4800);
+                    serialSetBaudRate(gpsPort, 9600);
                     gpsData.state_position++;
                     break;
                 case 1: // second run
                     // print the init string for the baudrate we want to be at
-                    serialPrint(gpsPort, "$PSRF100,1,115200,8,1,0*05\r\n");
+#if defined(USE_GPS_MTK)
```

Comment:
Isn't it better to send all three (based on USE_GPS) and let GPS decide?

Reply:
Removed USE_GPS_MTK and USE_GPS_SIRF

Reply:
My idea was to remove #else (and send setting setting for each defined model), but simply sending all is probably fine

Reply:
Yeah @ctzsnooze was asking for this.